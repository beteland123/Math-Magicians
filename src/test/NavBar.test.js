import { render } from '@testing-library/react';
import {
  MemoryRouter, Routes, Route,
} from 'react-router-dom';
import Navbar from '../components/NavBar';

describe('test navabar', () => {
  test('it should match the snapshoot', () => {
    const { container } = render(
      <MemoryRouter initialEntries={['/Navbar']}>
        <Routes>
          <Route path="/Navbar" element={<Navbar />} />
        </Routes>
      </MemoryRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
