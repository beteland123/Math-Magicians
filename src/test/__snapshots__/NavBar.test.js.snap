// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test navabar it should match the snapshoot 1`] = `
<div>
  <nav
    class="nav"
  >
    <h1>
      Math Magicians
    </h1>
    <div
      class="navDiv"
    >
      <a
        href="/"
      >
        Home
      </a>
      <a
        href="/Calculator"
      >
        Calculator
      </a>
      <a
        href="/Quote"
      >
        Quote
      </a>
    </div>
  </nav>
</div>
`;
